### ISSUE TITLE
Portal not retrieving data from VBO server

### ISSUE THREAD

**Issue by BenRio79** [VeeamHub/vbo365-rest-self-service#4]
Hi,

After a successful installation of the portal on a LAMP VM, I logged in as an Administrator and figured out that no data are being retrieved from my VBO server:

![image](https://user-images.githubusercontent.com/57013337/67614986-88125980-f7c6-11e9-9cf8-b0c37248f9c0.png)

![image](https://user-images.githubusercontent.com/57013337/67615091-04f20300-f7c8-11e9-83e8-c61a6ae0c146.png)

Also the login as a tenant with organization credentials fails, which is no doubt a consequence of the issue above. 

REST services and tenants authentication are enabled in VBO 3.

Is there anything related to REST I should install additionally on the WS2016 VBO3 server? I was not able to test locally if the VBO RESTful API is working properly so far on that machine.

Any ideas are welcome, Thanks!

Regards, Benoit

**Comment by nielsengelen** [comment]
Hi Benoit, can you verify if you are running the latest Veeam Explorers?

**Comment by BenRio79** [comment]
Hi Niels, 
Explorers were indeed up-to-date, but removing and installing them again solved the issue! Thank you for that ;-)  

However, I still cannot login as a tenant, with organization credentials. Error: Invalid credentials. 
Consider the following :
- VBO -> Options -> Authentication -> "Enable tenant authentication" enabled.
- Restoring a tenant from a remote Explorer using a CloudConnect Gateway does work. 

Any idea?
Thanks !

**Comment by nielsengelen** [comment]
You have to leverage the O365 account on this portal (so the account that has been added to the VBO365 console). Using the cloud connect account is not possible. Can you try if that works?

**Comment by BenRio79** [comment]
Yes, I always tried to login with O365 organization credentials at the self-service portal, with no success so far. And just like for the Explorers which I uninstalled and reinstalled to solve my previous issue, refreshing tenant organizations in VBO actually solved this login issue! Well... fair enough ;-)

Note that I mentioned CloudConnect since it was until today the way I used to retrieve O365 backups from a tenant: The design with VBR and VEX at the tenant side, and VBR + VBO at the service provider side, both connected through a CloudConnect gateway. Since adding a "VBO for service provider" store with O365 organization credentials from an Explorer worked out for my tenants, I wondered why this should not work with your portal... 

Case closed. Now I can get rid of the old and heavy design. Thank you very much!
